packages/bpk-component-calendar/README.md

```diff
@@ -174,6 +174,7 @@ withCalendarState(composeCalendar(
 | headerProps           | object               | false               | null             |
 | gridProps             | object               | false               | null             |
 | dateProps             | object               | false               | null             |
+| weekDayKey            | string               | false               | nameAbbr         |
```

Comment:
Could we list the valid options in the #weekDayKey section, and then link to it like this:

```suggestion
| weekDayKey            | see [weekDayKey](#weekDayKey)               | false               | nameAbbr         |
```